docs/client_reference.rst

```diff
@@ -0,0 +1,261 @@
+            If param is ``None``, :func:`asyncio.get_event_loop`
+            is used for getting default event loop, but we strongly
+            recommend to use explicit loops everywhere.
+            (optional)
+
+
+   :param request_class: Custom Request class implementation (optional)
+
+   :param response_class: Custom Response class implementation (optional)
+
+   :param dict cookies: Cookies to send with the request (optional)
+
+   :param dict headers: HTTP Headers to send with
+                        the request (optional)
+
+   :param aiohttp.helpers.BasicAuth auth: BasicAuth named tuple that represents
+                                          HTTP Basic Auth (optional)
+
+
+   .. py:method:: ClientSession.request(method, url, *, params=None, data=None, headers=None, auth=None, allow_redirects=True, max_redirects=10, encoding='utf-8', version=HttpVersion(major=1, minor=1), compress=None, chunked=None, expect100=False, read_until_eof=True)
+      :module: aiohttp.client
+
+      Performs an asyncronous http request. Returns a response object.
+
+      This method is a :ref:`coroutine <coroutine>`.
+
+      :param str method: HTTP method
+
+      :param str url: Request URL
+
+      :param dict params: Parameters to be sent in the query
+                          string of the new request (optional)
+
+      :param data: Dictionary, bytes, or file-like object to
+                   send in the body of the request (optional)
+
+      :param dict headers: HTTP Headers to send with
+                           the request (optional)
+
+      :param aiohttp.helpers.BasicAuth auth: BasicAuth named tuple that represents
+                                             HTTP Basic Auth (optional)
+
+      :param bool allow_redirects: If set to ``False``, do not follow redirects.
+                                   ``True`` by default (optional).
+
+      :param aiohttp.protocol.HttpVersion version: Request http version (optional)
+
+      :param bool compress: Set to ``True`` if request has to be compressed
+                            with deflate encoding.
+                            ``None`` by default (optional).
+
+      :param int chunked: Set to chunk size for chunked transfer encoding.
+                      ``None`` by default (optional).
+
+      :param bool expect100: Expect 100-continue response from server.
+                             ``False`` by default (optional).
+
+      :param bool read_until_eof: Read response until eof if response
+                                  does not have Content-Length header.
+                                  ``True`` by default (optional).
+
+
+
+
+   .. py:method:: ClientSession.get(url, *, allow_redirects=True, **kwargs)
+      :module: aiohttp.client
+
+      Perform a ``GET`` request.
+      In order to modify inner :func:`request<aiohttp.client.ClientSession.request>`
+      parameters, provide `kwargs`.
+
+      The function is a :ref:`coroutine <coroutine>`.
```

Comment:
I should noted that there are multiple places where method described as function. Sorry for not be clear on this. Please check the other methods as well.